personio-personnel-data-api-oa3.yaml

```diff
@@ -2107,6 +2107,7 @@ paths:
                             type: AbsenceType
                             attributes:
                               id: "9bba303f-0fbc-4514-9958-0befa21923fb"
+                              name: "Absence Type Name"
```

Comment:
Should this actually be the `example` key of the attribute?

Reply:
Sorry Simon I didn't get your question, but if I understand correctly this is a typical example sample of the response, but the definition of the response is in `HourlyAbsence` with the specs you mentioned

Reply:
Sorry, I realized this is part of the examples already, you can disregard this comment!